rdr_service/dao/genomics_dao.py

```diff
@@ -956,7 +958,7 @@ def to_client_json(self, result):
 
         client_json = {
             "participant_report_statuses": report_statuses,
-            "timestamp": result['date']
+            "timestamp": result['date'].replace(tzinfo=pytz.UTC)
```

Comment:
`result['date']` is timezone unaware so this works. But `replace` overwrites timezone data on a datetime object without changing anything else, even if it already has a timezone. So a date of 2020-05-01T03:00-06:00 would become 2020-05-01T03:00+00:00. I can't imagine this would ever see anything other than a timezone unaware `result['date']`. But _if_ there's ever a change in that, this could cause hard to detect bugs.

`pytz.utc.localize(result['date'])` functionally does the same as what you have, but it only accepts timezone unaware dates. So in the off chance anything changes about how the date is given to the dao, we'd see the unit tests failing and act accordingly.

This feels really nit-picky, but I thought I'd bring it up in case you thought it was a good idea.

Reply:
That's a good catch, I will make that fix. We are not expecting dates with timezone, but if that changes in the future, we would have a hard to find data issue on our hands. Better to localize than outright replace the timezone.